docs/rfc/ethereum_attestation.md

```diff
@@ -0,0 +1,126 @@
+The claims have no expiration date and don't need to be renewed.
+
+Every new claim invalidates previous ones made with the same account.
+To only revoke a claim without creating a new one, use ID `0`,
+which is guaranteed to not match any existing identity.
+
+Currently supported `version` values:
+- `1` - an `id` is a SHA-1 root hash. The excess high bytes are zeros, e.g. for hash
+`fb3102b74d7254eed7f18a31a3ba1ea946bb1a99` the passed `id` is
+`000000000000000000000000fb3102b74d7254eed7f18a31a3ba1ea946bb1a99`
+- `2` - an `id` is a SHA-256 root hash
+
+Everybody must use the same smart contract address.
+Claims sent to an address not known by the users won't be noticed by anyone and will be useless.
+
+## Creation of an attestation
+
+You need to perform 2 actions in any order:
+- Add or update an `ethereum` entry in your identity JSON.
+The entry's `account` must be your Ethereum address.
+It's higly recommended to set a reasonable expiration date as Ethereum claims don't expire.
+- Call `claim` in the `Claims` smart contract. The `id` must point to your link identity.
+
+## Discovery from an Ethereum address
+
+When you have an Ethereum address, you can find the claimed link ID using an Ethereum client.
+The example calls are based on the standard [client JSON RPC API][rpc] and should be exposed
+by your favorite Ethereum client library.
+It is important that the client must be trusted not to hide the events.
+
+- Use [getLogs][rpc-logs] to get the newest `Claimed` event filtered for the given ethereum address
+- Get the event's `transactionHash` field and use it to fetch the transaction which emitted it with
+[getTransactionByHash][rpc-tx]
+- Validate that the transaction signature matches its data and the ethereum address.
+For reference the signature payload content is listed [here][rpc-sign].
+- Read the link ID from the transaction data
+- Verify that the link ID claims back the Ethereum address,
+see [Discovery from a Link ID](#discovery-from-a-link-id)
+
+## Discovery from a Link ID
+
+When you have a link ID, you can find the claimed Ethereum address.
+Obtain the tip of its identity chain and read the ethereum address from the identity JSON
+`account` field in section `ethereum`, unless it's expired.
+You need to verify that the given Ethereum address claims back the link ID,
+see [Discovery from an Ethereum address](#discovery-from-an-ethereum-address).
+
+## Revocation of an attestation
+
+When your attestation for whatever reason is no longer valid,
+you should revoke it as soon as possible.
+Only one claim needs to be revoked to break the attestation,
+but to improve security you should revoke both sides if you can.
+
+To revoke a claim on Link, update and publish the identity JSON.
+You can change the claimed Ethereum address or remove the `ethereum` section altogether.
+
+To revoke a claim on Ethereum, call the `claim` function in `Claims` contract.
+You can claim a different link ID or an ID `0` to stop claiming anything.
```

Comment:
> to stop claiming anything.

could we rephrase so something like:

"to revoke any claim you may have."

Reply:
Fixed